File: 유진/4월2주차/[PGS]이상한 문자 만들기.js
function solution(s) {
  var answer = "";
  let string = s.split(" ");

  string.forEach((item, index) => {
    let list = [];
    let n_list = item.split("");
    n_list.forEach((item, index) => {
      if (index % 2 === 0) {
        // 자리가 짝수인 경우
        list.push(item.toUpperCase());
      } else {
        // 자리가 홀수인 경우
        list.push(item.toLowerCase());
      }
    });
    string[index] = list.join("");
  });
  answer = string.join(" ");

  return answer;
}

File: 은빈/자바/[PGS]이상한문자만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int idx = 0;
        
        for(int i = 0; i < s.length(); i++) {
            char ch = s.charAt(i);
            if(ch == ' ') {
                idx = 0;
                answer += ch;
            } else if(idx % 2 == 0) {
                idx++;
                answer += Character.toUpperCase(ch);
            } else {
                idx++;
                answer += Character.toLowerCase(ch);
            }
        }
        
        return answer;
    }
}